scoap3/articles/admin.py

```diff
@@ -314,6 +314,20 @@ def make_compliance_check(self, request, queryset):
             """,
         )
 
+    @admin.action(description="Mark as compliant")
+    def mark_as_compliant(self, request, queryset):
+        ids = []
+        for obj in queryset:
+            make_compliant.delay(obj.id)
```

Comment:
should not be a task.. it should just do `.update()` on the queryset list to update the `compliant` values